descriptive-link-5effbb.md

```diff
@@ -0,0 +1,237 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links in their context describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+
+authors:
+  - Carlos Duarte
+  - Marie Trudelle 
+  - Ramit Garg
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element with the [semantic role](#semantic-role) of [`link`](https://www.w3.org/TR/wai-aria/#link) or a role that inherits from the `link` role, that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible-name](#accessible-name) that does not only consist of [whitespace](#whitespace).
+
+## Expectation
+
+The [visible](#visible) [text content](#text-content) of the target element, together with its [programmatically determined link context](#programmatically-determined-link-context) describe the purpose of the link.
+
+**Note:** There may be situations where the purpose of the link is supposed to be unknown or obscured. In those situations, the success criteria will still be met, even if the link's purpose is not described by it.
```

Comment:
I'd rather see it as an Assumption.
Which would fit a nice pattern of "this rule assumes that we are not considering one of the exceptions allowed by WCAG".